packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -978,6 +978,7 @@ export class ToolbarAwareTabBar extends ScrollableTabBar {
     protected toolbar: TabBarToolbar | undefined;
     protected breadcrumbsContainer: HTMLElement;
     protected readonly breadcrumbsRenderer: BreadcrumbsRenderer;
+    protected dockPanel: TheiaDockPanel;
```

Comment:
Technically, this isn't defined from construction onward, though it should be defined by the time we're interested in it. Do we want the typing to reflect its late assignment?

```suggestion
    protected dockPanel?: TheiaDockPanel;
```